src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -353,7 +353,15 @@ public PrivilegesEvaluatorResponse evaluate(
             namedXContentRegistry
         );
 
+        final boolean serviceAccountUser = user.isServiceAccount();
         if (isClusterPerm(action0)) {
+            if (serviceAccountUser) {
+                presponse.missingPrivileges.add(action0);
+                presponse.allowed = false;
+                log.info("{} is a service account which has no access to cluster level permission of {}.", user, action0);
```

Comment:
nit: `{} is a service account which doesn't have access to cluster level permission: {}` Thoughts?

Reply:
fixed